orcid-persistence/src/main/resources/db-master.xml

```diff
@@ -214,5 +214,5 @@
     <include file="/db/updates/fix-null-relationship-on-funding-ext-ids.xml" />
     <include file="/db/updates/identifier-types/identifier-type-pdb.xml" />
     <include file="/db/updates/identifier-types/identifier-type-kuid.xml" />
-    <include file="/db/updates/identifier-types/identifier-type-lensid.xml" />
+    <include file="/db/updates/identifier-types/identifier-type-36-lensid.xml" />
```

Comment:
@TomDemeranville hey men! please see this change, it looks you changed the name of the lensid liquibase but forgot to update the db-master.